### DISCUSSION TITLE
Owner/Operator definition interpretation

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#22] [General]
Let's unpack the Owner/Operator definition in this discussion.  Here is the definition:

“owner” or “operator”, in relation to a virtual asset trading platform, means — 
(a)  the single entity or group which exerts management control over the platform; 
(b) where a single entity or group which exerts management control is not identifiable, either — 
(i) the entity under which the platform operates; or 
(ii) the entity under which the platform may contract with third parties; or 
(c) where an entity or group under paragraph (a) or (b) is not identifiable, the entity or group that provides the services offered by the platform to the users of the platform; 

Sub (a) is straightforward. What entities do you think are or could be captured under sub (b) and (c)?

**Comment by CaraProvenance** [comment]
@Aaron-XReg, it may be too simplistic, but a potential scenario, for B) be where a platform is trading under one name but registered under another, e.g. Alphabet Inc. ultimately controls Google's operations. Google trades but Alphabet controls? 

I would suggest that (c) is likely the section most suited to eek out in relation to Foundations.  As per earlier chat, Foundations are more obscurely structured than other corporate structures.  If there is no ultimate controlling party/decision maker etc., then to whome have the Foundation appointed/outsourced the responsibility for provision of platform service to, as Foundations are not typically operators or controllers in the provision of Financial Services. So perhaps to whomever they have engaged to provide such service?

**Comment by Insights24** [comment]
The definition is very broad, and sub(c) appears to be a very wide category (a catch all). Clarity may need to be provided in relation to same.  While the intention may be to align legislation with international standard setting recommendations of ultimately identifying "a responsible person", care must be exercised in terms of including persons who may not ultimately be "responsible".
